src/my-element.ts

```diff
@@ -175,6 +189,32 @@ export class MyElement extends LitElement {
   })
   quadrants = ['D', 'i', 'S', 'C']
 
+  constructor() {
```

Comment:
This is being used as a stand-in for loading in mock data. The constructor will probably be removed once there is a proper pipeline for loading in users.

Reply:
*definitely be removed